Cargo.toml

```diff
@@ -16,12 +16,11 @@ features = ["stm32g081", "rt"]
 default-target = "thumbv6m-none-eabi"
 
 [dependencies]
-cortex-m = "0.6.3"
-nb = "0.1.1"
-as-slice = "0.1.3"
+cortex-m = "0.7.1"
```

Comment:
If I remember correctly there was somekind of conflict if some deps are using 0.6 and others 0.7.
I think we should stay on 0.6 until this is resolved. It was discussed in matrix channel I do not have any better evidence on hand at the moment.

Reply:
`cortex-m 0.6` is forward compatible except for [CBP, ITM, MPU, NVIC, SCB](https://github.com/rust-embedded/cortex-m/issues/333), and there's no way to resolve that except update everything to `0.7`.

Reply:
Ok, lets move forward.